github/ci/prow-deploy/files/jobs/kubevirt/libguestfs-appliance/libguestfs-appliance-periodics.yaml

```diff
@@ -6,6 +6,9 @@ periodics:
   cron: "0 0 * * 0"
   annotations:
     testgrid-create-test-group: "false"
```

Comment:
We need to create a test-group if we want to use testgrid's email capqabilities, the changes required for that are:
* add a dashboard `libguestfs-appliance-periodics` to the base config here https://github.com/kubevirt/project-infra/blob/master/github/ci/testgrid/gen-config.yaml
* add an annotation referring to that test-group in the job like here https://github.com/kubevirt/project-infra/blob/master/github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml#L535 (instead of the current `testgrid-create-test-group: "false"`)

Reply:
Thanks for the review and the clarification. I'll update it Tomorrow morning (CEST time).